great_expectations/data_context/types/base.py

```diff
@@ -153,7 +153,6 @@ def to_yaml_str(self) -> str:
         """
         return object_to_yaml_str(obj=self.commented_map)
 
-    @public_api
```

Comment:
I am ok with removing these for now. It is unclear to me if that is the right decision. It seems like people may want to serialize these out and this provides a means to do that.